CHANGELOG.rst

```diff
@@ -8,6 +8,26 @@ GitHub issues are referenced, and can be viewed with hyperlinks on the `github r
 
 .. _`github releases page`: https://github.com/Libensemble/libensemble/releases
 
+Release 0.10.2
+--------------
+
+:Date: July 20, 2023
+
+* Fix issues with workflow dirs:
+  * Ensure relative paths are interpreted from where calling script is run. #1020
```

Comment:
Is it from "from where libEnsemble is being run" or "where the script is located"? (What if a user calls a script in a directory than where they are sitting?)

Reply:
A relative path is telling libEnsemble where the script is located.

Reply:
Going with "Ensure relative paths are interpreted from where libEnsemble is run"